tests/api/printf.cpp

```diff
@@ -128,17 +160,12 @@ TEST_F(WithCommandQueue, SimplePrintf) {
     ASSERT_STREQ(printf_buffer, message);
 }
 
-TEST_F(WithCommandQueue, TooLongPrintf) {
-    // each print takes 12 bytes (4 for the printf_id, and 2*4 for the 2 integer
-    // to print) + 4 for the byte written counter
-    auto cfg1 =
-        CLVK_CONFIG_SCOPED_OVERRIDE(printf_buffer_size, uint32_t, 28, true);
-
-    temp_folder_deletion temp;
-    stdoutFileName = getStdoutFileName(temp);
-
-    int fd;
-    ASSERT_TRUE(getStdout(fd));
+TEST_F(WithPrintfEnabled, TooLongPrintf) {
+    auto props = setup_arm_printf_test(44);
```

Comment:
Why `44`? it used to be `28` with a comment explaining why, but now it's another value that we don't know where it comes from.